components/brave_shields/browser/ad_block_subscription_service_manager.cc

```diff
@@ -308,6 +309,45 @@ void AdBlockSubscriptionServiceManager::RefreshSubscription(const GURL& sub_url,
   StartDownload(sub_url, true);
 }
 
+const char* DEFAULT_LISTS[] = {
+    "https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/"
+    "filters.txt",
```

Comment:
These come from https://github.com/brave/adblock-resources/blob/master/filter_lists/default.json. Hardcoding it like this is a bit awkward, and I'm open to suggestions on how to do it better. Ideally we could pull in the file from the other repo directly and have it templated into C++ somehow so it's always up to date, and maybe _only_ include it in builds where the service key is missing to shave off some binary size. Although the list is small and doesn't change too often, so it also may not be worth the extra complexity.

This list getting out of date on older installs isn't a concern - the logic using it could only ever run on the first browser launch.

Reply:
Very hacky, but one idea would be to add a quick & dirty GitHub action in `brave/adblock-resources` which pulls down this file and compares the contents of this variable with the JSON list. Auto-generating would be better of course, but having a check would at least help us catch the fact that the two lists may have drifted.

Reply:
Makes sense. In that case we should almost definitely put this list into a file of its own.